src/render/direct3d12/SDL_render_d3d12.c

```diff
@@ -52,16 +59,6 @@
 #define SDL_COMPOSE_ERROR(str) SDL_STRINGIFY_ARG(__FUNCTION__) ", " str
 #endif
 
-#ifdef __cplusplus
-#define SAFE_RELEASE(X) \
-    if (X) {            \
-        (X)->Release(); \
-        X = NULL;       \
-    }
-#define D3D_CALL(THIS, FUNC, ...)             (THIS)->FUNC(__VA_ARGS__)
-#define D3D_CALL_RET(THIS, FUNC, RETVAL, ...) *(RETVAL) = (THIS)->FUNC(__VA_ARGS__)
-#define D3D_GUID(X)                           (X)
```

Comment:
Unless I'm missing something, removal of this particular D3D_GUID macro will be a problem even when you're using CINTERFACE in c++ mode.

Reply:
You're right. I didn't notice this macro.